src/cascadia/TerminalSettingsModel/AllShortcutActions.h

```diff
@@ -23,58 +23,59 @@
 // case should be customized.
 
-#define ALL_SHORTCUT_ACTIONS             \
-    ON_ALL_ACTIONS(CopyText)             \
-    ON_ALL_ACTIONS(PasteText)            \
-    ON_ALL_ACTIONS(OpenNewTabDropdown)   \
-    ON_ALL_ACTIONS(DuplicateTab)         \
-    ON_ALL_ACTIONS(NewTab)               \
-    ON_ALL_ACTIONS(CloseWindow)          \
-    ON_ALL_ACTIONS(CloseTab)             \
-    ON_ALL_ACTIONS(ClosePane)            \
-    ON_ALL_ACTIONS(NextTab)              \
-    ON_ALL_ACTIONS(PrevTab)              \
-    ON_ALL_ACTIONS(SendInput)            \
-    ON_ALL_ACTIONS(SplitPane)            \
-    ON_ALL_ACTIONS(TogglePaneZoom)       \
-    ON_ALL_ACTIONS(SwitchToTab)          \
-    ON_ALL_ACTIONS(AdjustFontSize)       \
-    ON_ALL_ACTIONS(ResetFontSize)        \
-    ON_ALL_ACTIONS(ScrollUp)             \
-    ON_ALL_ACTIONS(ScrollDown)           \
-    ON_ALL_ACTIONS(ScrollUpPage)         \
-    ON_ALL_ACTIONS(ScrollDownPage)       \
-    ON_ALL_ACTIONS(ScrollToTop)          \
-    ON_ALL_ACTIONS(ScrollToBottom)       \
-    ON_ALL_ACTIONS(ResizePane)           \
-    ON_ALL_ACTIONS(MoveFocus)            \
-    ON_ALL_ACTIONS(Find)                 \
-    ON_ALL_ACTIONS(ToggleShaderEffects)  \
-    ON_ALL_ACTIONS(ToggleFocusMode)      \
-    ON_ALL_ACTIONS(ToggleFullscreen)     \
-    ON_ALL_ACTIONS(ToggleAlwaysOnTop)    \
-    ON_ALL_ACTIONS(OpenSettings)         \
-    ON_ALL_ACTIONS(SetColorScheme)       \
-    ON_ALL_ACTIONS(SetTabColor)          \
-    ON_ALL_ACTIONS(OpenTabColorPicker)   \
-    ON_ALL_ACTIONS(RenameTab)            \
-    ON_ALL_ACTIONS(OpenTabRenamer)       \
-    ON_ALL_ACTIONS(ExecuteCommandline)   \
-    ON_ALL_ACTIONS(ToggleCommandPalette) \
-    ON_ALL_ACTIONS(CloseOtherTabs)       \
-    ON_ALL_ACTIONS(CloseTabsAfter)       \
-    ON_ALL_ACTIONS(TabSearch)            \
-    ON_ALL_ACTIONS(MoveTab)              \
-    ON_ALL_ACTIONS(BreakIntoDebugger)    \
-    ON_ALL_ACTIONS(TogglePaneReadOnly)   \
-    ON_ALL_ACTIONS(FindMatch)            \
-    ON_ALL_ACTIONS(NewWindow)            \
-    ON_ALL_ACTIONS(IdentifyWindow)       \
-    ON_ALL_ACTIONS(IdentifyWindows)      \
-    ON_ALL_ACTIONS(RenameWindow)         \
-    ON_ALL_ACTIONS(OpenWindowRenamer)    \
-    ON_ALL_ACTIONS(GlobalSummon)         \
-    ON_ALL_ACTIONS(QuakeMode)            \
+#define ALL_SHORTCUT_ACTIONS               \
+    ON_ALL_ACTIONS(CopyText)               \
+    ON_ALL_ACTIONS(PasteText)              \
+    ON_ALL_ACTIONS(OpenNewTabDropdown)     \
+    ON_ALL_ACTIONS(DuplicateTab)           \
+    ON_ALL_ACTIONS(NewTab)                 \
+    ON_ALL_ACTIONS(CloseWindow)            \
+    ON_ALL_ACTIONS(CloseTab)               \
+    ON_ALL_ACTIONS(ClosePane)              \
+    ON_ALL_ACTIONS(NextTab)                \
+    ON_ALL_ACTIONS(PrevTab)                \
+    ON_ALL_ACTIONS(SendInput)              \
+    ON_ALL_ACTIONS(SplitPane)              \
+    ON_ALL_ACTIONS(ToggleSplitOrientation) \

```

Comment:
You know, auto code formatting is nice, but sometimes it makes the diffs a bit annoying